File: app/src/main/java/ru/prsolution/winstrike/mvp/apimodels/Offer.java

package ru.prsolution.winstrike.mvp.apimodels;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Offer implements Serializable
{

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("cost")
    @Expose
    private Integer cost;
    @SerializedName("public_id")
    @Expose
    private String publicId;
    @SerializedName("create_at")
    @Expose
    private String createAt;
    private final static long serialVersionUID = 4831832608759182598L;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getCost() {
        return cost;
    }

    public void setCost(Integer cost) {
        this.cost = cost;
    }

    public String getPublicId() {
        return publicId;
    }

    public void setPublicId(String publicId) {
        this.publicId = publicId;
    }

    public String getCreateAt() {
        return createAt;
    }

    public void setCreateAt(String createAt) {
        this.createAt = createAt;
    }

}

File: app/src/main/kotlin/ru/prsolution/winstrike/datasource/model/Offer.kt
package ru.prsolution.winstrike.datasource.model

import java.io.Serializable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Offer : Serializable {

	@SerializedName("name")
	@Expose
	var name: String? = null
	@SerializedName("cost")
	@Expose
	var cost: Int? = null
	@SerializedName("public_id")
	@Expose
	var publicId: String? = null
	@SerializedName("create_at")
	@Expose
	var createAt: String? = null

	companion object {
		private const val serialVersionUID = 4831832608759182598L
	}

}
